CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All Notable changes to `digipolisgent/drupal_theme_gent-base`.
 
+## [Unreleased]
+* DTGB-665: Refactored pages to only show layouts. This shouldn't affect any
```

Comment:
Not sure if this belongs in the `gent-base` CHANGELOG, since only styleguide was affected by these changes.